### ISSUE TITLE
cls are you use in bce loss

### ISSUE THREAD

**Issue by RockMan1966** [hamidriasat/UNet-3-Plus#3]
Hi author, did you use binary cross entropy for supervised classification in the loss, I noticed this is mentioned in the author's paper.

**Comment by hamidriasat** [comment]
Hi,
Because I am not giving training support for `UNet 3+ Classification Guided Module`, so binary cross entropy loss is not employed.
